src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1344,4 +1344,7 @@
   <data name="PreferIsEmptyOverCountTitle" xml:space="preserve">
     <value>Prefer IsEmpty over Count</value>
   </data>
+  <data name="UseAsSpanInsteadOfRangeIndexerCodeFixTitle" xml:space="preserve">
+    <value>Use {0} instead of Range-based indexers on {1}</value>
+  </data>
```

Comment:
```suggestion
  <data name="UseAsSpanInsteadOfRangeIndexerOnAnArrayCodeFixTitle" xml:space="preserve">
    <value>Use {0} instead of Range-based indexers on an array</value>
  <data name="UseAsSpanInsteadOfRangeIndexerOnAStringCodeFixTitle" xml:space="preserve">
    <value>Use {0} instead of Range-based indexers on a string</value>
  </data>
  </data>
```